lib/camp3/resource.rb

```diff
@@ -62,9 +62,8 @@ def self.create(hash)
     end
 
     def self.detect_type(url)
-      Camp3.logger.debug "Request URL: #{url}" 
       case url
-      when /#{Camp3.api_endpoint}\/projects\/\d+\.json/
+      when /#{Camp3.base_api_endpoint}\/\d+\/projects\/\d+\.json/
```

Comment:
Facilitates running the *request_spec.rb* since it won't require configuring the basecamp account a priori in the `Configuration` class